lib/xdr/transactions/operation_result_tr.ex

```diff
@@ -1,8 +1,21 @@
-defmodule StellarBase.XDR.OperationInnerResult do
+defmodule StellarBase.XDR.OperationResultTr do
   @moduledoc """
-  Representation of Stellar `OperationInnerResult` type.
+  Automatically generated by xdrgen
+  DO NOT EDIT or your changes may be overwritten
+
+  Target implementation: elixir_xdr at https://hex.pm/packages/elixir_xdr
+
+  Representation of Stellar `OperationResultTr` type.
   """
-  alias StellarBase.XDR.{InvokeHostFunctionResult, OperationType}
+
+  @behaviour XDR.Declaration
+
+  alias StellarBase.XDR.{
+    BumpFootprintExpirationResult,
+    RestoreFootprintResult,
+    InvokeHostFunctionResult,
+    OperationType
+  }
```

Comment:
`BumpFootprintExpirationResult`, `RestoreFootprintResult`, and `InvokeHostFunctionResult` modules should be placed under the namespace `StellarBase.XDR.Operations`